docs/ldml/tr35.md

```diff
@@ -1474,9 +1474,11 @@ If a language has more than one script in customary modern use, then the CLDR fi
 lang
 lang_script
 lang_script_region
-lang_region (aliases to lang_script_region)
+lang_region (aliases to lang_script_region based on likely subtags)
 ```
 
+Note: When the requested locale is `lang_script_region` and `lang_region` implies the `script` according to likely subtags, then resource lookup should start at `lang_region`. This applies whether or not the language has more than one script in customary modern use. For example, `zh-Hant-TW` should start at `zh-TW` (since `zh-TW` implies `Hant`), and `de-Latn-LI` should start at `de-LI` (since `de` implies `Latn` and `de-LI` does not have its own entry in likely subtags).
```

Comment:
Add another note:

Note: Item lookup also requires use of the parentLocales, as described in <a href="#Parent_Locales">Parent Locales</a>.

Reply:
There is already a statement below the example block that says:

> _Both the resource bundle inheritance and the inherited item inheritance use the parentLocale data, where available, instead of simple truncation._